scripts/gulp/tasks/docs.ts

```diff
@@ -199,12 +199,20 @@ task('serve:docs:hot', async () => {
     app.use(
       WebpackDevMiddleware(compiler, {
         publicPath: webpackConfig.output.publicPath,
-        contentBase: paths.docsSrc(),
-        hot: process.env.NODE_ENV !== 'production',
-        quiet: false,
-        noInfo: true, // must be quite for hot middleware to show overlay
-        lazy: false,
-        stats: config.compiler_stats,
+
+        // contentBase: paths.docsSrc(),
+
+        // hot: process.env.NODE_ENV !== 'production',
+
+        // quiet: false,
+
+        // noInfo: true, // must be quite for hot middleware to show overlay
+
+        // the lazy option was removed without replacement
+        // lazy: false,
+
+        // the stats option was removed, the default value of the stats option is taken from the value of the stats option from the configuration (webpack.config.js)
+        // stats: config.compiler_stats,
```

Comment:
???

Reply:
All of these options have been removed from WebpackDevMiddleware; I can remove the commented out values, but I went through docs and changelog with these one by one.

Here's an example error we see:

```
[08:21:59] ValidationError: Invalid options object. Dev Middleware has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'stats'. These properties are valid:
   object { mimeTypes?, writeToDisk?, methods?, headers?, publicPath?, serverSideRender?, outputFileSystem?, index? }
```

Options docs here:
https://github.com/webpack/webpack-dev-middleware#options

Changelog here:
https://github.com/webpack/webpack-dev-middleware/blob/master/CHANGELOG.md#breaking-changes

No mention in options or of removal in changelog:
`contentBase`, `hot`, `quiet`, `noInfo` 

Changelog mentions removal:
`lazy` - the lazy option was removed without replacement.
`stats` - the stats option was removed, the default value of the stats option is taken from the value of the stats option from the configuration (webpack.config.js)

I could certainly use help here validating, because I don't have background in why this is used and what should be affected. If you see new options which should be added, let me know.

Reply:
I've removed the commented out options but would appreciate if you could double check these.